ros2service/ros2service/api/__init__.py

```diff
@@ -43,3 +44,21 @@ def __call__(self, prefix, parsed_args, **kwargs):
                 node=node,
                 include_hidden_services=getattr(
                     parsed_args, self.include_hidden_services_key))
+
+
+class ServiceTypeCompleter:
+    """Callable returning an existing service type or all service types."""
```

Comment:
Nit: "an existing service type" I would recommend the documentation that it's returning the service type matching the passed service_name or all types (same for topics below)

Reply:
That sounds good but exceed the one line rule for the docblock :wink: